# xiaohu_robot::NodeControlMessage
string next=next

string state
string control